scripts/get_version.sh

```diff
@@ -28,4 +28,4 @@
 
 set -e
 
-grep -oP "PROJECT_VERSION \"?\K[0-9.]+(?=\")?" "$(dirname "$0")/../CMakeLists.txt"
+sed -n -e 's/^\s*set(PROJECT_VERSION "\([0-9.]\+\)")\s*$/\1/p' "$(dirname "$0")/../CMakeLists.txt"
```

Comment:
Like @aarlt already said, doesn't work for me as well, but his suggestion works. So the line should be:
```suggestion
sed -n -e 's/^\s*set(PROJECT_VERSION "\(.*\)")$/\1/p' "$(dirname "$0")/../CMakeLists.txt"
```